### DISCUSSION TITLE
How do I set an upper limit on the camera rotation speed?

### DISCUSSION THREAD

**Discussion by shine1919** [Sixze/ALS-Refactored#341] [Q&A] [answered]
How do I set an upper limit on the camera rotation speed?

**Comment by ameaninglessname** [comment, marked as answer]
You could clamp the `InputValue` of `IA_Als_Look` by adding a custom modifier, or simply clamp the value in something like `AAlsCharacterExample::Input_OnLook`

**Reply by shine1919** [reply]
Thank you.
